config/settings.d/dhcp_native_ms.yml.example

```diff
@@ -0,0 +1,12 @@
+---
+#
+# Configuration file for Windows-specific 'netsh' dhcp provider
+#
+# Uncomment the following line if you wish to disable dynamic DNS updates for all hosts that
+# the smart proxy creates. This will slightly slow the host creation process but will ensure
+# that the DHCP server will not create or delete DNS entries on behalf of these clients. If
+# you can turn this feature off at the scope level then that would be a better solution.
+#
+# :disable_ddns: true
```

Comment:
Can you define a default value for ```disable_ddns``` in https://github.com/theforeman/smart-proxy/blob/develop/modules/dhcp_native_ms/dhcp_native_ms_plugin.rb too?